src/configuration/routes/cache.md

```diff
@@ -23,58 +42,80 @@ https://{default}/:
         default_ttl: 60
 ```
 
-In this example, requests will be cached based on the URI, the `Accept` header, `Accept-Language` header, and `X-Language-Locale` header; Any response that lacks a `Cache-Control` header will be cached for 60 seconds; and the presence of any cookie in the response will disable caching of that response.
+## Cache configuration properties
 
-## Cache Per Route
+### `enabled`
+
+Turns the cache on or off for a route.
 
-If you need fine-grained caching, you can set up caching rules for
-several routes separately.
+**Type:** Boolean
 
-Here is an example:
+**Values**
+* `true`: enable the cache for this route
+* `false`: disable the cache for this route [default]
+
+### `headers`
+
+Adds specific header fields to the cache key, enabling caching of separate responses for those headers.
+
+For example, if the `headers` key is the following, Platform.sh will cache a different response for each value of the `Accept` HTTP request header:
```

Comment:
Changed to `Platform.sh will cache a different response for each value of the `Accept` HTTP request header "only"`.